modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must specify the projects associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same project as the `ServiceMeshControlPlane` resource and name it `default`.
+
+[WARNING]
+====
+If Container Network Interface (CNI) is enabled, manual sidecar injection will work, but pods will not be able to communicate with the control plane unless those pods are a part of the `ServiceMeshMemberRoll` resource.
```

Comment:
This should be "If the Istio CNI (Container Network Interface) _plugin_ is enabled, ...".

Also: instead of "are a part of the SMMR resource", it might be clearer to say "are specified/listed in the SMMR resource".

Reply:
Fixed.